packages/kit/src/runtime/server/page/index.js

```diff
@@ -102,7 +103,7 @@ export async function render_page(event, route, page, options, state, resolve_op
 		}
 
 		const should_prerender_data = nodes.some((node) => node?.server);
-		const data_pathname = `${event.url.pathname.replace(/\/$/, '')}/__data.json`;
+		const data_pathname = `${event.url.pathname.replace(/\/$/, '')}/__data.js`;
```

Comment:
This could reuse `DATA_SUFFIX`

Reply:
358d0ceb323f9c53a2bfc6ef1acd9f742eadf1b5